### ISSUE TITLE
Emformer pretrain

### ISSUE THREAD

**Issue by dykim3** [pytorch/audio#2499]
### 🚀 The feature

Is there any seperated pretraining process in Emformer implementation? 
the time, frequency masking is seen in training. so can I assume there aren't any pretraining step in acoustic model?

### Motivation, pitch

^^

**Comment by xiaohui-zhang** [comment]
Hi @dykim3 , No. the model was trained from scratch on transcribed data. 
the time and frequency masking is data augmentation techniques. It's not relevant to pre-training.
